### DISCUSSION TITLE
划词翻译主题配色

### DISCUSSION THREAD

**Discussion by lmk123** [hcfyapp/crx-selection-translate#931] [讨论]
划词翻译目前只支持深浅两套配色，我想给划词翻译添加更多配色，例如 v6.x 时的绿色，但是我自己几乎毫无设计经验，搭配出来的配色总感觉不好看，所以希望有设计经验的用户能提供配色方案，我会选择一些添加进划词翻译当中。

这是我目前搭配的绿色主题：

![image](https://user-images.githubusercontent.com/5035625/111250776-de489c80-8648-11eb-9605-b20b5535dcfb.png)

我感觉白色的部分有点刺眼，但不知道应该替换成什么颜色比较好。

**Comment by GH01** [comment]
我只有些使用阅历，你看看有没用：
（我贴图不如你自己试一下）

https://pc.qq.com/detail/3/detail_163.html
QQ（国际版）的做法：
在其[界面管理器]窗口中，点选一个颜色（或图片）作为背景，然后调节右下角的滑条（改变主窗前景的透明度），同时眼睛观察主窗，直至满意。
▲我觉得这种设计简单而且适合你的需求。

Vivaldi的做法（汉化不准确，请切到英文版查看原文）：
在[Settings: Themes]中：
设置几个关键色：Background | Foreground | Highlight | Accent
还可以对 Contrast/Opacity/Saturation 做进一步调节。
▲然后你观察整个界面，可以取到比较和谐的深浅配色。
（Edge好像也是类似做法）

**Reply by lmk123** [reply]
设计真难 :joy:

我打算加个选项，允许用户自定义翻译面板的 CSS，然后再加个链接让用户分享他们设计的面板，其它用户选一个觉得好看的应用上去。

**Comment by zhhaojie** [comment]
> 例如 v6.x 时的绿色，但是我自己几乎毫无设计经验，搭配出来的配色总感觉不好看。

难怪UI看起来那么没有设计感觉。 我回头看看公司有没有前端的同学有空，帮助调一个样式出来。
